### ISSUE TITLE
CSS collision in \assets\sass\modules\_registration.scss with the content class

### ISSUE THREAD

**Issue by rodiojp** [okta/okta-signin-widget#266]
With the most recent npm version 1.13.0, there is an issue with ".registration-container .content" in _registration.scss .  To fix this issue .content might be renamed into .registration-content class, because there is a duplication with .content class in _okta-footer.scss
```
.registration-container {
  margin-top: 30px;
  
  **.content {**
    padding: 30px 42px;
    -moz-transition: padding-top 0.4s;
    -webkit-transition: padding-top 0.4s;
    transition: padding-top 0.4s;
    border-top: 1px solid #ddd;
    margin-top: 5px;
  }

```
the issue is here:
https://jsfiddle.net/rodiojp/hc017r1p/2/

Temporary fix is here:
https://jsfiddle.net/rodiojp/65xf6zbb/3/

**Comment by mraible** [comment]
Can you try version 2.1.0? It's available on npm and via CDN at http://cdn.jsdelivr.net/npm/@okta/okta-signin-widget@2.1.0/dist/js/okta-sign-in.min.js

**Comment by rodiojp** [comment]
The issue is with CSS only. The javascript code works.
![image](https://user-images.githubusercontent.com/30329871/28471012-a0023996-6e09-11e7-9be9-2c097b63b8da.png)

**Comment by mraible** [comment]
2.1.0 contains a fix for https://github.com/okta/okta-signin-widget/issues/126, so the Okta Sign-In Widget's CSS is not global. I believe this will fix your issue.

**Comment by StateBarofArizona** [comment]
2.1.0 does seem to fix the issue but is missing a couple images such as the checkbox image.

**Comment by mraible** [comment]
@StateBarofArizona I fixed the checkbox issue in https://github.com/okta/okta-signin-widget/pull/248. In the meantime, you might be able to work around it by copying the image to the expected location in your app.

**Comment by hor-kanchan-okta** [comment]
Should be fixed in #284 . Release version 2.3.0
